package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@itslearning/protomorph",
     "description": "Shared build config for frontend applications",
-    "version": "11.0.0-beta.7",
```

Comment:
See previous comment. 

I will update to the next beta release when it's ready.

Reply:
Ah, alright. Thanks! Fixed now